Vagrantfile

```diff
@@ -313,9 +313,9 @@ Vagrant.configure(2) do |config|
 
```

Comment:
I only see a slight fix for Vagrantfile changes introduced in windows branch - why should they be in master now?